theforeman.org/pipelines/vars/candlepin/nightly.groovy

```diff
@@ -1,8 +1,12 @@
 def candlepin_version = 'nightly'
-def candlepin_distros = ['el8']
 def packaging_branch = 'rpm/develop'
+def candlepin_distros = [
+    'el8',
+    'el9'
+]
 def pipelines = [
     'candlepin': [
-        'centos8-stream'
+        'centos8-stream',
+        'centos9-stream'
```

Comment:
This is currently unused (as we do not yet have CP tests), but would start failing once we do, or does this PR mean we also have el9 packages now, not only repos?

Reply:
https://copr.fedorainfracloud.org/coprs/g/theforeman/candlepin-nightly-staging/build/6888982/ seems like there are EL9 builds 🤘

Reply:
Means both -- I tested this before hand with nightly https://yum.theforeman.org/candlepin/nightly/el9/x86_64/

Reply:
And working puppet module tests -- https://github.com/theforeman/puppet-candlepin/pull/251